library/Backends/llvm/test/Test/Golden.hs

```diff
@@ -158,5 +158,10 @@ toEither (Feedback.Success app a) = Right (app, a)
 toEither (Feedback.Fail failure) = Left failure
 
 -- Running by hand example
--- x <- Feedback.runFeedbackT (compile "../../../test/examples/positive/llvm/MainApply.ju")
--- toEither x
+printCompile :: FilePath -> IO ()
+printCompile filePath = do
+  x <- Feedback.runFeedbackT (compile filePath)
+  case toEither x of
+    Right (_, str) ->
+      putStrLn str
+    Left err -> print err
```

Comment:
is this part of another branch that got caught up?

Reply:
no, I just decided to change the comment to a function to just print the given code

Reply:
Might as well leave it in!